.github/workflows/generate-html-for-pull-request.yaml

```diff
@@ -16,33 +16,53 @@ jobs:
         uses: actions/checkout@v2
```

Comment:
Seems to be a newer version available? No need to change, but should probably be checked at some point.

Reply:
Good point! I think that applies to other workflows as well, would probably be good to go through all of them and update any actions that have newer versions.